src/main/java/wolox/training/models/Book.java

```diff
@@ -48,6 +54,16 @@
     @NotNull
     private String isbn;
 
+    /**
+     * Represents the foreign association between book and user
+     */
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
So if there is a many-to-one relationship between Book and User that means that a User can have multiple books, but each Book only can have a relation with only one user. Check the specification of the card that ask you that a Book can be in the book collection of more than one User.